src/DistortableCollection.js

```diff
@@ -218,7 +218,7 @@ L.DistortableCollection = L.FeatureGroup.extend({
 
     json.images = json.images.reverse();
     json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
-
+    json.images.map(each=> localStorage.setItem('locations', JSON.stringify(each.nodes)));
```

Comment:
Here is your code, i see! Actually I wonder if this code should be outside the `generateExportJson()` method. Should there be a method like `collection.saveToLocalStorage()` instead? What do you think?